src/components/layout/__tests__/Col.test.tsx

```diff
@@ -9,4 +10,10 @@ describe('Col', () => {
     expect(component).toMatchSnapshot('Col');
     component.unmount();
   });
+
+  it('is accessible', async () => {
+    const render = () => '<Col width="full"></Col>'
```

Comment:
This would just return a string - instead we'd want to mount the component using enzyme, then pass the HTML output to axe. This would look a bit like this:
```jsx
const component = shallow(<Col width="full"></Col>);
expect(await axe(component.html())).toHaveNoViolations();
  ```

Reply:
Also it might be worth using a different example to show accessibility testing - the `Col` component is just a div with a couple of classes, and isn't too complicated. Maybe apply this to a fully mounted and rendered `<Checkboxes>` component, with a few boxes?